app/views/hours/_form.html.slim

```diff
@@ -0,0 +1,35 @@
+= simple_form_for [@houred, @hour] do |f|
+  = simple_error_notice f
+
+  .row
+    .col-xs-12
+      - if action_name ==  'new'
+        = f.association :assignment, collection: @assignments_clients
+      - else
+        = f.hidden_field :assignment_id, value: @hour.assignment.id
+
+  .row
+    .col-xs-12
+        = f.hidden_field :volunteer_id, value: params[:volunteer_id] || @hour.volunteer.id
+
+  .row
+    .col-xs-12
+      = f.input :meeting_date, include_blank: true, start_year: Date.today.year - 1, end_year: Date.today.year
+
+  .row
+    .col-xs-12.col-md-6
+      = f.input :hours, collection: 1..8
```

Comment:
Maybe move `1..8` and `0..59` into constants, e.g. `Hour::HOUR_RANGE` and `Hour::MINUTE_RANGE`?